### ISSUE TITLE
BRCFilteredTableViewController draws search query over results

### ISSUE THREAD

**Issue by OnlyInAmerica** [Burning-Man-Earth/iBurn-iOS#42]
When scrolling through search results the query is drawn over the results.

![caption](http://i.imgur.com/4fF0tCO.png)

**Comment by chrisballinger** [comment]
Yeah... dealing with the search bars is such a pain without causing weird internal Apple framework problems.
